vulnerabilities/tests/test_openssl.py

```diff
@@ -20,466 +20,48 @@
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1e",
-                        ),
-                    ),
-                    AffectedPackage(
-                        vulnerable_package=PackageURL(
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1b",
-                        ),
-                        patched_package=PackageURL(
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1e",
-                        ),
-                    ),
-                    AffectedPackage(
-                        vulnerable_package=PackageURL(
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1c",
-                        ),
-                        patched_package=PackageURL(
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1e",
-                        ),
-                    ),
-                    AffectedPackage(
-                        vulnerable_package=PackageURL(
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1d",
-                        ),
-                        patched_package=PackageURL(
-                            type="generic",
-                            namespace=None,
-                            name="openssl",
-                            version="1.1.1e",
-                        ),
-                    ),
-                ],
-                references=[
-                    Reference(
-                        reference_id="",
-                        url="https://github.com/openssl/openssl/commit/419102400a2811582a7a3d4a4e317d72e5ce0a8f",
-                        severities=[],
-                    ),
-                    Reference(
-                        reference_id="",
-                        url="https://github.com/openssl/openssl/commit/f1c5eea8a817075d31e43f5876993c6710238c98",
-                        severities=[],
-                    ),
-                ],
-            ),
-        ]
-
-        found_advisories = OpenSSLImporter.to_advisories(data)
-
-        found_advisories = list(map(Advisory.normalized, found_advisories))
-        expected_advisories = list(map(Advisory.normalized, expected_advisories))
-        assert sorted(found_advisories) == sorted(expected_advisories)
+def parse_advisory_data() -> Iterable[AdvisoryData]:
```

Comment:
Should there be another tests that uses the main XML parsing code from `openssl.py` too?

Reply:
No not.